kubernetes/.swagger-codegen-ignore

```diff
@@ -1,8 +1,8 @@
 .gitignore
 git_push.sh
-# Remove this once swagger-codegen 2.2.3 is released and we update. 
+# Remove this once swagger-codegen 2.3.0 is released and we update. 
```

Comment:
Do we use the HEAD of swagger-codegen for this PR? In that case, why we still have this line?